app/views/books/show.html.erb

```diff
@@ -1,14 +1,14 @@
-<div class="container p-3 text-center">
-  <div class="book-cover">
-    <img src=<%= @book.cover_image %> alt="book cover" class="mb-3">
+<div class="container p-3">
+  <div class="container book-cover book-info">
+    <img src=<%= @book.cover_image %> alt="book cover" class="mb-3 img-center">
     <h2><%= @book.title %></h2>
     <p>by <%= @book.author %></p>
-    <span>Description:</span>
+    <%# <span>Description:</span> %>
```

Comment:
You can just delete the things. Please do not leave to much clutter. Version control is taking care of it, we can always find it in earlier versions.

Reply:
Ok!